### PR TITLE
adding tracking to borrower-info pieces of loancard.

### PR DESCRIPTION
**Pull request by BoulderBrains**
Adding tracking to borrower name and "read more" text on loan cards.

### CODE REVIEW

**Comment by mcstover** [comment]
@BoulderBrains Need to update both of these to be a string literal `` and then use ${id} for that var.

**Comment by mcstover** [comment]
I'm not sure if that will work... Usually the `` wrap the full string like you have in the :to attribute.

How did it render the attribute with this in place?

File: src/components/LoanCards/BorrowerInfo.vue [hunk 1]
```diff
@@ -1,13 +1,22 @@
 <template>
 	<div class="borrower-info-wrapper">
-		<router-link :to="`/lend/${ id }`" class="name">{{ name }}</router-link>
+		<router-link
+			:to="`/lend/${ id }`"
+			class="name"
+			v-kv-track-event="['Lending', 'click-Read more', 'Name', id, 'true']">
```
Comment (jedlin-kiva) [inline src/components/LoanCards/BorrowerInfo.vue]: I think this one should be something like 'click-borrower name' so we can distinguish which elements are getting clicked (I'll change the link around the borrower photo to 'click-borrower photo') for the same reason.

**Comment by mcstover** [comment]
@jedlin-kiva I think it's already using the other parameter to make that distinction.

That said, ultimately, any tracking we add that is being duplicated from existing tracking, should match exactly, the parameters in use so that any reports will function as they do today.

**Comment by jedlin-kiva** [comment]
@mcstover - yeah, I figured that out after I made the comment, so deleted the comment. :0

**Comment by BoulderBrains** [comment]
Thank you both @jedlin-kiva & @mcstover

**Pull request merged**